src/avram/save_operation.cr

```diff
@@ -123,9 +123,17 @@ abstract class Avram::SaveOperation(T) < Avram::Operation
       end
 
       def set_{{ attribute[:name] }}_from_param(_value)
-        parse_result = {{ attribute[:type] }}::Lucky.parse(_value)
+        {% if attribute[:type].is_a?(Generic) %}
+          # HACK:
+          # We have to pass an array her, but what's `_value`?
+          # If `_value` is `"true"`, then we pass `["true"]` which can be parsed, but
+          # what if we need to pass in multiple values? I don't know that params handles that currently
```

Comment:
I think this is spot on. Right now there is no way for params to return anything other than a single value. I'd say that right now that is ok and can be handled later. People can manually parse JSON responses, or if it is URL encoded I'd say it's probably best to use a `String` attribute and manually parse it into an array.

Typically most sites that use these have to do some kind of param parsing/manipulation before save anyway so I think this is ok and can be improved later